airflow/providers/google/cloud/example_dags/example_bigquery_queries.py

```diff
@@ -40,9 +40,12 @@
 PROJECT_ID = os.environ.get("GCP_PROJECT_ID", "example-project")
 DATASET_NAME = os.environ.get("GCP_BIGQUERY_DATASET_NAME", "test_dataset")
 LOCATION = "southamerica-east1"
+BUCKET_1 = os.environ.get("GCP_GCS_BUCKET_1", "test-gcs-example-bucket")
```

Comment:
```suggestion
BUCKET_1 = os.environ.get("GCP_BIGQUERY_QUERIES_BUCKET", "test-biqgquery-queries-bucket")
```
I found this env variable is used in `airflow/providers/google/cloud/example_dags/example_gcs.py`.
I looked on the other system tests and env vars names are like: `GCP_DATAPROC_BUCKET` or `GCP_TEXT_TO_SPEECH_BUCKET`